server/routes.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, request, Response
+import db
+
+app = Flask(__name__)
+
+#connect to the database
+conn = db.init_db()
+
+@app.route('/tags', methods=['GET', 'POST'])
+def tags():
+    if request.method == 'GET':
+        res = db.read_tags(conn)
+        return res
+    if request.method == 'POST':
+        body = request.get_json()
+        res = db.create_tag(conn, body)
+        return res
+
+@app.route('/tags/<int:tag_id>', methods=['GET', 'DELETE'])
+def tags_by_id(tag_id):
```

Comment:
If this returns a single tag, we should probably name this as singular `tag_by_id`